### DISCUSSION TITLE
Dynamic API & Client-Id based on Environment

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#9588] [General - Components / Directives / etc] [answered]
Dear All , 

I have 3 environments Local , Dev & Prod . Each environment has its own values for API & Client-Id . How  I can configure Quasar in such a way that if I run quasar dev -Local  , it should read the values from .env.local file , If I run the command like quasar dev -Dev it should read the values from .env.dev file . If I build through quasar build it should refer to .env.prod file.

Please help me to achieve this.

**Comment by metalsadman** [comment, marked as answer]
https://quasar.dev/quasar-cli/quasar-conf-js#using-dotenv

**Reply by jmriyaz84** [reply]
Thanks for your help , 

I went to this link , https://www.npmjs.com/package/@quasar/quasar-app-extension-dotenv ,
it says  **dotenv package no longer supported. Use 'build->env' in quasar.conf.js_** to do same thing.

From quasar.conf file it is possible to declare the values for various environments like Local , Dev & Prod.
